account_chatter/views/account_account_views.xml

```diff
@@ -1,16 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
 <odoo>
-    <record id="view_account_form_inherit" model="ir.ui.view">
-        <field name="name">account.account.form.inherit</field>
+
+    <record id="view_account_form" model="ir.ui.view">
+        <field name="name">account.account.form.inherit.chatter</field>
```

Comment:
Maybe adding `account` before the `chatter` since more things besides the chatter could be modified later on  🤔